### ISSUE TITLE
Directory Opus preview ingetration

### ISSUE THREAD

**Issue by OdinZhang** [ccseer/Seer#420]
Can seer support Directory Opus preview panel, so I needn't to type space to preview.

**Comment by ccseer** [comment]
I can't do anything inside DOpus

**Comment by OdinZhang** [comment]
Yes, but directory opus can support other apps as the preview provider. Such as office, sumatra pdf, powertoys. Called preview handlers, I think this feature will support ms explorer too.
![image](https://github.com/ccseer/Seer/assets/25601078/8b6b479c-37d7-43c0-87d3-ac18115a292a)

**Comment by Darthagnon** [comment]
Related? #291

**Comment by OdinZhang** [comment]
> Related? #291

no, I want seer support dopus preview panel, not perform same as ms explorer

**Comment by titmouse84** [comment]
While I don't vibe with the tone of the previous comment, I agree with the sentiment. This would be a very nice addition.

**Comment by ccseer** [comment]
It's possible nowadays. 
The coming version is focusing on implementing changes based on full AI review. The version after that should be a full UI polish also based on AI.  Then this feature could be added at a later stage.
